src/ice/agent.rs

```diff
@@ -1333,9 +1333,11 @@ impl IceAgent {
 
         let reply = StunMessage::reply(req.trans_id, req.source);
 
-        debug!(
+        trace!(
```

Comment:
If this is changed to trace, perhaps the sending side should be changed too?

Reply:
good catch, updated